README.md

```diff
@@ -19,7 +19,7 @@
 
 `go get -u github.com/pion/mediadevices`
 
-## Usage
+### Usage
```

Comment:
Hierarchy of the sections are changed to
```
## Install
### Usage
### More Examples
### Available Media Inputs
### Available Codecs
#### Video Codecs
##### x264
### FAQ
#### Failed to find the best driver that fits the constraints
### Roadmap
### Community
### Contributing
### License
```
but I don't think these '###' sections are subsections of `## Install`

Reply:
Oh, I missed the install section. Thanks. I fixed it now.

All Pion repos are using level-3 sections in the readme as the highest level (`###`).